recipe/meta.yaml

```diff
@@ -49,7 +49,7 @@ outputs:
         - kiwisolver >=1.3.1
         - numpy >=1.23,<3.0a0
         - packaging >=20.0
-        - pillow >=8
+        - pillow >=9
```

Comment:
Unless I am mistaken, it looks like the pillow requirement is still `>=8` 
https://github.com/matplotlib/matplotlib/blob/v3.10.0/pyproject.toml#L40

Reply:
I was accidentally looking at the main branch 🤦